### PR TITLE
[RUM][REPLAY] Disclose data-* attributes in mask mode

### PR DESCRIPTION
**Pull request by acorretti**
### What does this PR do? What is the motivation?

Disclose that Session Replay will also mask `data-*` attributes when the `mask` mode is used.

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

**Review by BenoitZugmeyer — COMMENTED** [review]
It's missing the [1]

**Review by BenoitZugmeyer — APPROVED** [review]

File: content/en/real_user_monitoring/session_replay/privacy_options.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Masks most form fields such as inputs, text areas, and checkbox values while rec
 
 ### Mask mode
 
-Setting `defaultPrivacyLevel` to `mask` mode will mask all HTML text, user input, images, and links. Text on your application is replaced with `X`, rendering the page into a wireframe.
+Setting `defaultPrivacyLevel` to `mask` mode will mask all HTML text, user input, images, links and [`data-*` attributes][1]. Text on your application is replaced with `X`, rendering the page into a wireframe.
```
Comment (maycmlee) [inline content/en/real_user_monitoring/session_replay/privacy_options.md:62]:
```suggestion
Setting `defaultPrivacyLevel` to `mask` mode masks all HTML text, user input, images, links and [`data-*` attributes][1]. Text on your application is replaced with `X`, rendering the page into a wireframe.
```

**Review by maycmlee — COMMENTED** [review]
Just a small edit

File: content/en/real_user_monitoring/session_replay/privacy_options.md (hunk 1 above)
Comment (acorretti) [reply content/en/real_user_monitoring/session_replay/privacy_options.md:62]: Feel free to commit this change and merge yourself, I didn't edit pre-existing copy.

**Review by amortemousque — APPROVED** [review]

**Pull request merged**